prisma/schema.prisma

```diff
@@ -0,0 +1,32 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "sqlite"
+  url      = "file:./dev.db"
+}
+
+model Measurement {
+  date                       DateTime @unique
+  componentAFlowKgH          Float
+  componentBFlowKgH          Float
+  coolantTemperatureC        Float
+  reactorOutletTemperatureC  Float
+  reactorHotspotTemperatureC Float
+  yieldPercent               Float
+  productFlowTonsPerDay      Float
+  notes                      Note[]
+}
+
+model Note {
+  id          Int         @id @default(autoincrement())
+  date        DateTime
+  metrics     String // comma separeted list of metric field names
```

Comment:
Problems I see are:
- it's not space-effective
- it's  difficult to reuse because `metrics` field it opaque to the database
- changing name of metric requires painful migration